content/working_with/manager/using-secrets.md

```diff
@@ -63,6 +63,17 @@ For more commands, see [secrets command line]({{< relref "cli/orch_cli/secrets.m
 
 ![Create Secret]( /images/manager/create_secret_dialog.png )
 
+### Creating a secret with Secret Provider from the {{< param cfy_console_name >}}
```

Comment:
Maybe to refer to terms convention used in this page better would be to have something like:

```suggestion
### Creating remotely provided secret from the {{< param cfy_console_name >}}
```